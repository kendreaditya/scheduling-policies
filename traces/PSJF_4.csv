1,0,10000
2,0,100
3,0,1
4,0,5
5,0,10
6,100,100
7,100,200
8,100,300
9,10000,1
10,10000,2
