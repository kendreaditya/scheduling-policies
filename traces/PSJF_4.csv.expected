1,10719
2,116
3,1
4,6
5,16
6,216
7,416
8,716
9,10001
10,10003
